docs/category-overview-pages/accessing-netdata-dashboards.md

```diff
@@ -1,3 +1,28 @@
 # Accessing Netdata Dashboards
 
-This section contains documentation on how you can access the Netdata Agent's dashboards, and the Netdata Cloud's dashboards.
\ No newline at end of file
+This section contains documentation on how you can access the Netdata dashboards, which are the same both for the Agent and Cloud.
+
+A user accessing the Netdata dashboard **through the Cloud** will always be presented with the latest Netdata dashboard version.
```

Comment:
A user accessing the Netdata dashboard **from the Cloud** will always be presented with the latest Netdata dashboard version.